frontend/components/icons/Success.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { COLORS, Colors } from "styles/var/colors";
```

Comment:
This seems a bit confusing to me. What's the difference between `COLORS` and `Colors`?

Reply:
one is a type and the other is the mapping. I'll change the names to make it more. clear